core/services/keystore/keys/ocr2key/export.go

```diff
@@ -35,13 +35,16 @@ func FromEncryptedJSON(keyJSON []byte, password string) (KeyBundle, error) {
 	}
 	switch export.ChainType {
 	case chaintype.EVM:
-		key := mustNewEVMKeyFromRaw(rawKey)
+		key := mustNewKeyFromRaw[*evmKeyring](rawKey, &evmKeyring{})
```

Comment:
Did you try removing the explicit type parameter? It looks like it can be inferred, or if not we should be able to tweak so that it can.
```suggestion
		key := mustNewKeyFromRaw(rawKey, &evmKeyring{})
```